### ISSUE TITLE
NTR: Ultrasound, CT or MRI: inferior vena caval obstruction (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8925]
New term request Ultrasound, CT or MRI: inferior vena caval obstruction (elthat_100822190753): Compression; Membranous obstruction of inferior vena cava (MOVC); stenosis; atresia; atretic  UMLS: 3496580

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000417
- label: Inferior vena cava obstruction
- parents: [Abnormal inferior vena cava morphology (HP:0025576)](https://hpo.jax.org/app/browse/term/HP:0025576)
- pmid: PMID:32809720
- def: Physical invasion or compression of the inferior vena cava by a pathological process or by thrombosis within the vein itself.
- comment: Inferior vena cava syndrome (IVCS) iis a sequence of signs and symptoms that refers to obstruction or compression of the inferior vena cava (IVC). Symptoms and signs of IVCS result from a reduced venous return to the heart, as well as a pooling of blood in the IVC, which causes hypotension, tachycardia, lower extremity edema, elevated liver enzymes, end-organ failures and, hypoxia, altered mental status, and death. IVCS is less frequently found in comparison with the superior vena cava syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8925
- synonyms:
